controllers/csi/gc/logs.go

```diff
@@ -0,0 +1,124 @@
+package csigc
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	dtcsi "github.com/Dynatrace/dynatrace-operator/controllers/csi"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+)
+
+const (
+	maxLogFolderSize    = 300000
+	maxAmountOfLogFiles = 1000
+	maxLogAge           = 14 * 24 * time.Hour
+)
+
+func (gc *CSIGarbageCollector) runLogGarbageCollection(versionReferences []os.FileInfo, tenantUUID string) error {
+	fs := &afero.Afero{Fs: gc.fs}
+	gcPodUIDs, err := gc.getPodUIDs(fs, versionReferences, tenantUUID)
```

Comment:
What Pod UIDs are returned? Method and variable could use a better name.

Reply:
Renamed